### PR TITLE
Added tests for team/models

### PR DESCRIPTION
**Pull request by dotiurin**
Added tests for 'del_update' attribute of 'update' method
Added tests for 'del_users' method of Team instance
Fixed 'get_by_id()' to 'objects.get()'

## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTP-132)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/team/testmodel.py [hunk 1]
```diff
@@ -22,12 +22,31 @@ def setUp(self):
...
+                                     password='niceday')
+            custom_user.set_password('niceday')
+            custom_user.save()
+
             custom_user = CustomUser(id=101,
                                      first_name='john',
                                      last_name='doe',
                                      middle_name='eric',
                                      email='email',
                                      password='123456')
```
Comment (lhalam) [inline eventually/tests/unittests/team/testmodel.py]: remove this line

File: eventually/tests/unittests/team/testmodel.py [hunk 2]
```diff
@@ -22,12 +22,31 @@ def setUp(self):
...
+                                     password='donuts')
+            custom_user.set_password('donuts')
+            custom_user.save()
+
+            custom_user = CustomUser(id=401,
+                                     first_name='Ned',
+                                     last_name='Flanders',
+                                     middle_name='HeyDiddlyHo',
+                                     email='member3@gmail.com',
+                                     password='niceday')
```
Comment (lhalam) [inline eventually/tests/unittests/team/testmodel.py]: remove this line

File: eventually/tests/unittests/team/testmodel.py [hunk 3]
```diff
@@ -22,12 +22,31 @@ def setUp(self):
         with mock.patch('django.utils.timezone.now') as mock_time:
             mock_time.return_value = TEST_TIME
 
+            custom_user = CustomUser(id=301,
+                                     first_name='Homer',
+                                     last_name='Simpson',
+                                     middle_name='Jay',
+                                     email='member2@gmail.com',
+                                     password='donuts')
```
Comment (lhalam) [inline eventually/tests/unittests/team/testmodel.py]: remove this line

File: eventually/tests/unittests/team/testmodel.py [hunk 4]
```diff
@@ -141,7 +160,7 @@ def test_team_all_update(self):
         Test for updating all attributes.
         """
 
-        actual_team = Team.get_by_id(101)
+        actual_team = Team.objects.get(id=101)
         new_owner = CustomUser.objects.create(id=201,
                                               email='exp@gmail.com',
                                               password='123')
```
Comment (lhalam) [inline eventually/tests/unittests/team/testmodel.py]: use set_password

File: eventually/tests/unittests/team/testmodel.py (hunk 3 above)
Comment (dotiurin) [reply eventually/tests/unittests/team/testmodel.py]: Done

File: eventually/tests/unittests/team/testmodel.py (hunk 2 above)
Comment (dotiurin) [reply eventually/tests/unittests/team/testmodel.py]: Done

File: eventually/tests/unittests/team/testmodel.py (hunk 1 above)
Comment (dotiurin) [reply eventually/tests/unittests/team/testmodel.py]: Done

File: eventually/tests/unittests/team/testmodel.py (hunk 4 above)
Comment (dotiurin) [reply eventually/tests/unittests/team/testmodel.py]: Done

**Review by lhalam — APPROVED** [review]

**Pull request merged**